src/ToEnum.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Filter;
+
+use BackedEnum;
+use Laminas\Filter\Exception\RuntimeException;
+use Laminas\Stdlib\ArrayUtils;
+use Traversable;
+use UnitEnum;
+
+use function is_array;
+use function is_int;
+use function is_string;
+use function is_subclass_of;
+
+/**
+ * @psalm-type Options array{enum: class-string<UnitEnum>}
+ * @extends AbstractFilter<Options>
+ */
+class ToEnum extends AbstractFilter
+{
+    /** @var Options */
+    protected $options = [
```

Comment:
The psalm errors detected here should be similar to https://github.com/laminas/laminas-validator/pull/168

Perhaps polyfilling with a `.phpstub` file in Psalm could suffice.